gdx/src/com/badlogic/gdx/graphics/g2d/GlyphLayout.java

```diff
@@ -170,8 +169,10 @@ else if (targetWidth <= font.data.spaceWidth) //
 									wrapIndex = i - 1;
 								}
 								GlyphRun next;
-								if (wrapIndex == 0)
+								if (wrapIndex == 0) {
 									next = run; // No wrap index, move entire run to next line.
+									run.width = 0;
+								}
```

Comment:
The run width was fixed recently. Can you please merge the latest `master` into your branch, fix conflicts, commit, and push?

Reply:
Ok, I saw it later :(. Thanks